recipe/bld.bat

```diff
@@ -1,10 +1,10 @@
-
 set OPENSSL_ROOT_DIR=%LIBRARY_PREFIX%
 
+set "CMAKE_GENERATOR=NMake Makefiles JOM"
```

Comment:
What is the reason to move away from `ninja`?

Reply:
I was following what they did at Anaconda since they have a working build. This is the reason for the changes. I reverted this one. Let's see what happens